ocs_ci/helpers/dr_helpers.py

```diff
@@ -2097,3 +2097,91 @@ def wait_for_vrg_state(
         )
         logger.info(error_msg)
         raise TimeoutExpiredError(error_msg)
+
+
+def validate_storage_cluster_peer_state():
+    """
+    Validate Storage cluster peer state
+
+    Raises:
+        TimeoutExpiredError: incase storage cluster peer state is not reached 'Peered' state.
+
+    """
+    restore_index = config.cur_index
+    managed_clusters = get_non_acm_cluster_config()
+    for cluster in managed_clusters:
+        index = cluster.MULTICLUSTER["multicluster_index"]
+        config.switch_ctx(index)
+        logger.info("Validating Storage Cluster Peer status")
+        sample = TimeoutSampler(
+            timeout=300,
+            sleep=5,
+            func=check_storage_cluster_peer_state,
+        )
+        if not sample.wait_for_func_status(result=True):
+            error_msg = (
+                "Storage cluster peer status does not have expected values within the time "
+                f"limit on cluster {cluster.ENV_DATA['cluster_name']}"
+            )
+            logger.error(error_msg)
+            raise TimeoutExpiredError(error_msg)
+    config.switch_ctx(restore_index)
+
+
+def check_storage_cluster_peer_state():
```

Comment:
Why do we need 2 functions for peer state validation?

Reply:
validate_storage_cluster_peer_state validates the correct state of managed cluster where other one gets the bool value of individual cluster ( might be used in -ve scenarios to check the individual cluster )